flink/v1.14/build.gradle

```diff
@@ -67,6 +67,7 @@ project(':iceberg-flink:iceberg-flink-1.14') {
       exclude group: 'org.apache.hive', module: 'hive-storage-api'
     }
 
+    testImplementation "org.apache.spark:spark-sql_2.12:3.2.0"
```

Comment:
Seems like this dependency was added for writing the timestamp as int96 in the unit test,  but in fact we apache flink's ParquetRowDataWriter support writing a timestamp_with_local_time_zone into an INT96.  So I will suggest to use the flink parquet writer rather than the spark parquet writer. (It's strange for me to introduce a spark module in in the flink module).

Reply:
I actually prefer using the Spark module, unless Flink natively supports writing INT96 timestamps to Parquet. The benefit of using the Spark module is that support has been around for a long time and is relatively trusted to produce correct INT96 timestamp values.